biogeochem/EDPhysiologyMod.F90

```diff
@@ -840,7 +853,14 @@ subroutine seed_germination( currentSite, currentPatch )
 
     do p = 1,numpft
        currentPatch%seed_germination(p) =  min(currentSite%seed_bank(p) * &
-             EDPftvarcon_inst%germination_timescale(p),max_germination)
+             EDPftvarcon_inst%germination_timescale(p),max_germination)     
+       !set the germination only under the growing season...c.xu
+       if (EDPftvarcon_inst%season_decid(p) == 1.and.currentSite%status == 1)then 
```

Comment:
good idea!

Reply:
This updates resolves some seasonal cycle component for grass at Delaware but I feel this could be good improvement for temperate/boreal forests....but will not impact tropical forest.